apps/daimo-mobile/src/view/screen/SendScreen.tsx

```diff
@@ -76,51 +79,116 @@ function Scan({ hide }: { hide: () => void }) {
 
 function CancelRow({ title, hide }: { title: string; hide: () => void }) {
   return (
-    <View style={styles.headerRow}>
-      <TextBold>{title}</TextBold>
-      <ButtonSmall onPress={hide}>
-        <Octicons name="x" size={16} color="gray" />
-      </ButtonSmall>
+    <View>
+      <View style={ss.spacer.h64} />
+      <View style={styles.headerRow}>
+        <View style={ss.spacer.w32} />
+        <TextSmall>{title}</TextSmall>
+        <ButtonSmall onPress={hide}>
+          <Octicons name="x" size={16} color="gray" />
+        </ButtonSmall>
+      </View>
     </View>
   );
 }
 
 function Search() {
   const [prefix, setPrefix] = useState("");
-  const results = useRecipientSearch(prefix.trim().toLowerCase());
+  const res = useRecipientSearch(prefix.trim().toLowerCase());
 
   return (
     <>
       <InputBig icon="search" value={prefix} onChange={setPrefix} />
-      {results.map((r) => (
-        <Result key={r.account.addr} recipient={r} />
+      {res.error && <ErrorRow error={res.error} />}
+      {res.recipients.map((r) => (
+        <RecipientRow key={r.account.addr} recipient={r} />
       ))}
+      {res.isSearching && res.recipients.length === 0 && (
+        <TextCenter>
+          <TextSmall>No results</TextSmall>
+        </TextCenter>
+      )}
     </>
   );
 }
 
-function Result({ recipient }: { recipient: Recipient }) {
+function ErrorRow({ error }: { error: { message: string } }) {
+  return (
+    <TextError>
+      <TextCenter>{error.message}</TextCenter>
+    </TextError>
+  );
+}
+
+function RecipientRow({ recipient }: { recipient: Recipient }) {
   const nav = useNav();
   const pay = useCallback(() => nav.setParams({ recipient }), []);
   return <ButtonBig title={recipient.account.name} onPress={pay} />;
 }
 
 function SendPayment({ recipient }: { recipient: Recipient }) {
-  const [amount, setAmount] = useState(0);
+  const [dollars, setDollars] = useState(0);
 
   const nav = useNav();
   const hide = useCallback(() => nav.setParams({ recipient: undefined }), []);
 
-  const send = useCallback(() => {
-    console.log("TODO send");
-  }, []);
+  const [account] = useAccount();
+  assert(account != null);
+
+  const { status, message, exec } = useSendTransaction(
+    account.enclaveKeyName,
+    recipient.account.addr,
+    dollars
+  );
+
+  // TODO: load estimated fees
```

Comment:
ah this was the intention for the `dryRun` parameter in @daimo/userop but I just realised that still asks for a signature: should change that behaviour in a seperate PR and use that